cost_analysis_standalone.py

```
@@ -572,6 +725,11 @@ def find_gpu_acquisition_price(price_df, hardware_model, price_colname):
     price_df = price_df.sort_values(by='Price date').dropna(subset=[price_colname])
     # Search for the best price closest to release date    
     matching_prices = price_df[price_df['Hardware model'].str.contains(gpu_hardware_alias)]
+
+    if matching_prices.empty:
+        print(f"Could not find any prices for '{hardware_model}' in the price data.")
+        return None
```

@robirahman, should we raise an exception here?

---

I'll replace the print with an exception tomorrow.